bundles/org.openhab.core.ephemeris/src/main/java/org/openhab/core/ephemeris/internal/EphemerisManagerImpl.java

```diff
@@ -107,6 +107,9 @@ public EphemerisManagerImpl(final @Reference LocaleProvider localeProvider, fina
         this.localeProvider = localeProvider;
         bundle = bundleContext.getBundle();
 
+        // Default weekend dayset
+        addDayset(CONFIG_DAYSET_WEEKEND, List.of(DayOfWeek.SATURDAY, DayOfWeek.SUNDAY));
```

Comment:
Is it still possible to customize the weekend with this change?

Not everyone has their weekends on these days: https://en.wikipedia.org/wiki/Workweek_and_weekend

Reply:
Of course, it is a default. In case, the parameter is defined in the configuration, this default is overwritten. It is what happens if you update the setting in Main UI (ephemeris settings). I tested it.

This default matches the default value that was defined for this parameter:
https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.ephemeris/src/main/resources/OH-INF/config/ephemeris.xml#L11

Reply:
Nice to know everyone can still have their own weekends! :wink: